File: com/bridgelab/algorithm/Anagram.java
package com.bridgelab.algorithm;

import java.util.Scanner;

import com.bridgelab.utility.Utility;

public class Anagram {

	public static void main(String args[]) {
		Utility u = new Utility();
		String first = u.getNext("Enter first string : ");
		String second = u.getNext("Enter second string : ");
		if(u.isAnagram(first, second)) System.out.println("String is anagram.");
		else System.out.println("String is not anagram.");
	}
}

File: Month1/algorithm/Anagram.py
from util.Utility import is_anagram


def main():
    str1 = input("Enter first string : ")
    str2 = input("Enter second string : ")

    if is_anagram(str1.strip(), str2.strip()):
        print("String is anagram.")
    else:
        print("String is not anagram.")


if __name__ == '__main__':
    main()
